### PR TITLE
Update changelog/package.json for 3.4.0 release

### PR DESCRIPTION
**Pull request by Blackbaud-SteveBrush**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,7 @@
+# 3.4.0 (2020-04-08)
+
+- Updated the colorpicker component to implement the affix and overlay services. [#74](https://github.com/blackbaud/skyux-lookup/pull/74)
```
Comment (Blackbaud-DeniseCisneros) [inline CHANGELOG.md:3]: Do affix and overlay need to be within tick marks? If not, looks good to me.

Comment (Blackbaud-DeniseCisneros) [reply CHANGELOG.md:3]: Wait...this says colorpicker. Is that correct?

Comment (Blackbaud-SteveBrush) [reply CHANGELOG.md]: Oops! Copy-pasted from the other changelog. (The services do not need the tick marks.)

Comment (blackbaud-johnly) [reply CHANGELOG.md]: Doesn't seem like it. This is the lookup repo, and colorpicker is not one of the components there.

**Comment by Blackbaud-BobbyEarl** [comment]
Preview build at [https://host.nxt.blackbaud.com/skyux-lookup?_sp.spa=pr-release-3.4.0](https://host.nxt.blackbaud.com/skyux-lookup?_sp.spa=pr-release-3.4.0)

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,3 +1,7 @@
+# 3.4.0 (2020-04-08)
+
+- Updated the autocomplete and lookup components to implement the affix and overlay services. [#74](https://github.com/blackbaud/skyux-lookup/pull/74)
```
Comment (blackbaud-johnly) [inline CHANGELOG.md:3]: Does the lookup component actually get these services? I see them imported into the autocomplete module, but I don't see a corresponding import for lookup.

**Pull request merged**